### PR TITLE
Install vonage-panel for grafana

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Files will be installed in
/var/lib/grafana/plugins/Vonage-Grafana_Status_panel directory

Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , @cloudbehl ,  Please review

File: tendrl-monitoring-integration.spec [hunk 1]
```diff
@@ -25,6 +26,8 @@ Python module for Tendrl to create a new dashboard in Grafana
 
 %prep
 %setup
+unzip %SOURCE1
```
Comment (nthomas-redhat) [inline tendrl-monitoring-integration.spec]: We need to build this plugin as a subpackage as in https://github.com/ceph/cephmetrics/blob/master/cephmetrics.spec.in#L33
package name should be tendrl-grafana-plugins

File: tendrl-monitoring-integration.spec [hunk 2]
```diff
@@ -25,6 +26,8 @@ Python module for Tendrl to create a new dashboard in Grafana
...
+mv -f Vonage* Vonage-Grafana_Status_panel
```
Comment (nthomas-redhat) [inline tendrl-monitoring-integration.spec]: @cloudbehl , please confirm the folder name is ok

**Comment by rishubhjain** [comment]
@TimothyAsirJeyasing @r0h4n Tested on local system, need to test this via installing package from copr.
@r0h4n Please review and merge patch for further testing

**Pull request merged**